Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformationManager.java

```diff
@@ -209,4 +209,138 @@ public ITransformedNetwork getSearchTransformedNetwork(String projectId, String
 
         return networkList;
     }
+    
+    
+    @Override
+    public ITransformedNetwork getSearchTransformedNetwork(List<String> projectIds, String conceptId)
+        throws QuadrigaStorageException {
+    	// get the transformed network and search for the concept id
+        List<INetwork> networkList = new ArrayList<>();
+        
+        for(String projectId : projectIds){
+        	List<INetwork> network = getNetworkList(projectId);
```

Comment:
your variable names should reflect their type, so if it's a list it should be plural in this case.